thrift/src/main/thrift/atoms/qanda.thrift

```diff
@@ -0,0 +1,19 @@
+namespace * contentatom.qanda
+namespace java com.gu.contentatom.thrift.atom.qanda
+#@namespace scala com.gu.contentatom.thrift.atom.qanda
+
+include "entity.thrift"
+include "../shared.thrift"
+
+struct QAndAAtom {
+  1: optional string typeLabel
+  2: optional list<string> tags
+  3: optional Image eventImage
+  4: required list<QAndAItem> items
+  5: optional string sidenote
+}
+
+struct QAndAItem {
```

Comment:
Ask Nathan about how/if this links to story questions and whether we plan on producing them in response to story questions.